StackExchange.Redis/StackExchange/Redis/GeoEntry.cs

```diff
@@ -0,0 +1,144 @@
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace StackExchange.Redis
+{
+
+    /// <summary>
+    /// Describes a GeoEntry element with the corresponding value
+    /// GeoEntries are stored in redis as SortedSetEntries
+    /// </summary>
+    public struct GeoEntry : IEquatable<GeoEntry>, IEquatable<SortedSetEntry>, IComparable, IComparable<GeoEntry>, IComparable<SortedSetEntry>
+    {
+        internal readonly RedisValue element;
+        internal readonly double score;
+
+        /// <summary>
+        /// Initializes a GeoEntry value
+        /// </summary>
+        public GeoEntry(RedisValue element,double score)
+        {
+            this.element = element;
+            this.score = score;
+        }
+
+        /// <summary>
+        /// The unique element stored in the sorted set
+        /// </summary>
+        public RedisValue Element => element;
+
+        /// <summary>
+        /// The score against the element
+        /// </summary>
+        public double Score => score;
```

Comment:
@wjdavis5 indeed, but the meaning of that value is opaque and not really exposed; the outside caller talks in terms of `longitude` and `latitude`